presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1833,7 +1833,9 @@ core::PlanFragment VeloxQueryPlanConverterBase::toVeloxQueryPlan(
       case protocol::SystemPartitioning::FIXED: {
         switch (systemPartitioningHandle->function) {
           case protocol::SystemPartitionFunction::ROUND_ROBIN: {
-            auto numPartitions = partitioningScheme.bucketToPartition->size();
+            auto numPartitions = partitioningScheme.bucketToPartition != nullptr
+                ? partitioningScheme.bucketToPartition->size()
+                : 1;
```

Comment:
I needed to add these back in. The `bucketToPartiton` field is optional and a fragment from a CREATE TABLE query does not populate them, causing this to fail with a NPE. With this check, then this fragments passes conversion

Reply:
how does it work when the plan that gets sent for execution?  Is it no longer missing at that point?  What does it look like then?

Reply:
I'm not sure, I didn't examine the output. I can run it again and post it here.

Reply:
@BryanCutler : I'm surprised that CREATE TABLE needs any actions at the worker at all. Can you provide the fragments you tested with ?